mmdet/utils/util_distribution.py

```diff
@@ -0,0 +1,59 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+import torch
+from mmcv.parallel import MMDataParallel, MMDistributedDataParallel
+
+dp_factory = {
+        'cuda' : MMDataParallel
+    }
+
+ddp_factory = {
+        'cuda' : MMDistributedDataParallel
+    }
+
+def build_dp(model, device = 'cuda', device_ids = 0):
+    """build DataParallel module by device type.
+
+    if device is cuda, return a MMDataParallel model; if device is mlu,
+    return a MLUDataParallel model.
+    """
+    assert device in ['cuda', 'mlu'], "Only available for cuda or mlu devices."
```

Comment:
We supported CPU training in PR https://github.com/open-mmlab/mmdetection/pull/7016
This change breaks this feature.

Reply:
fixed